js/modules/k6/k6_test.go

```diff
@@ -136,32 +138,42 @@ func TestGroup(t *testing.T) {
 	})
 }
 
-func TestCheck(t *testing.T) {
+func checkTestRuntime(t testing.TB, ctxs ...*context.Context) (
```

Comment:
Like 90% of the changes here are so the test is better structured .... I probably can pull some of those out in a separate commit, if we aren't going to go with this design or if the time allows

Reply:
separate PR/commit sounds good :+1:

Reply:
https://github.com/k6io/k6/pull/2094